### PR TITLE
MockExec (plugin class) => ExecHandleBuilder (internal Gradle class)

### PR DESCRIPTION
**Pull request by brunobowden**
- Logging of workingDir in projectExec

Even though this uses an internal Gradle class, I think it is a better trade off than the time and risks of re-implementing the class within the plugin.

### CODE REVIEW

**Comment by brunobowden** [comment]
@confile, @advayDev1 - PTAL

**Comment by confile** [comment]
LGTM

File: src/main/groovy/com/github/j2objccontrib/j2objcgradle/tasks/Utils.groovy [hunk 1]
```diff
@@ -299,51 +299,46 @@ class Utils {
...
-            return
+    static String projectExecLog(
+            ExecSpec execSpec, ByteArrayOutputStream stdout, ByteArrayOutputStream stderr,
+            boolean execSucceeded, Exception exception) {
+        // Add command line and stderr to make the error message more useful
+        // Chain to the original ExecException for complete stack trace
+        String msg = ''
+        // The command line can be long, so highlight more important details below
+        if (execSucceeded) {
+            msg += 'Command Line Succeeded:\n'
```
Comment (advayDev1) [inline src/main/groovy/com/github/j2objccontrib/j2objcgradle/tasks/Utils.groovy]: why init to empty and add here?

**Comment by advayDev1** [comment]
lgtm

File: src/main/groovy/com/github/j2objccontrib/j2objcgradle/tasks/Utils.groovy (hunk 1 above)
Comment (brunobowden) [inline src/main/groovy/com/github/j2objccontrib/j2objcgradle/tasks/Utils.groovy]: Fixed

**Pull request merged**